ui_testing/pages/order_page.py

```diff
@@ -363,8 +364,8 @@ def get_suborder_table(self):
         self.info(' Get suborder table list.')
         self.base_selenium.click(element='order:suborder_list')
 
-    def create_new_suborder(self, material_type='', article_name='', test_plan='', test_unit='',
-                            add_new_suborder_btn='order:add_new_item'):
+    def create_new_suborder(self, material_type='', article_name='', test_plans=[''], test_unit='',
```

Comment:
You need to see all usages again as master updated now, msy be any merged test case use this function withvtest unit string not list